.ci/pytorch/test.sh

```diff
@@ -328,6 +328,26 @@ test_h100_distributed() {
   assert_git_not_dirty
 }
 
+test_check_unimplemented_calls() {
+  # Get the changed files from the PR that match torch/_dynamo/**/*.py pattern
+  DYNAMO_PY_FILES=$(find torch/_dynamo/ -name "*.py" | tr '\n' ' ')
+
+  echo "Checking unimplemented_v2 calls in: $DYNAMO_PY_FILES"
+
+  if ! python tools/dynamo/gb_id_mapping.py check --files "$DYNAMO_PY_FILES" --registry-path tools/dynamo/graph_break_registry.json; then
```

Comment:
Why not move logic for discovering fiels directly into the script?
Also, do we really want to ship it as part of torch package?